api/src/main/java/com/messagebird/MessageBirdClient.java

```diff
@@ -43,7 +43,9 @@
      * can, however, override this behaviour by providing absolute URLs
      * ourselves.
      */
-    private static final String CONVERSATIONS_BASE_URL = "https://conversations.messagebird.com/v1";
```

Comment:
why it's not static final anymore?

Reply:
or final? it should be final

Reply:
Well is just looks weird that there is a constant variable which is not constant.

Reply:
Ok, changing it

Reply:
Made it better